FinalProject/gs-rest-service-master/complete/src/main/java/services/UserService.java

```diff
@@ -0,0 +1,19 @@
+package services;
+
+import java.sql.SQLException;
+import java.util.List;
+
+import bo.User;
+
+public interface UserService {

```

Comment:
Minor: it's not necessary to set access modifiers in method here because always are public in an interface.